pcp/PCPMetric.c

```diff
@@ -178,3 +178,21 @@ bool PCPMetric_fetch(struct timeval* timestamp) {
       *timestamp = pcp->result->timestamp;
    return true;
 }
+
+unsigned int PCPMetric_lookupInDom(const char* metric) {
+   pmID pmid;
+   pmDesc desc;
+   int sts;
+
+   sts = pmLookupName(1, &metric, &pmid);
+   if (sts < 0) {
+      return 0;
+   }
+
+   sts = pmLookupDesc(pmid, &desc);
+   if (sts < 0) {
+      return 0;
+   }
+
+   return desc.indom;
+}
```

Comment:
@smalinux I think we don't need this function - we have already done a descriptor lookup of each metric during initialization, and they're stashed off the main Platform structure for PCP.  It's better to use the cached values since pmLookupName and pmLookupDesc here involve round-trips to pmcd.  I'll send through some small cleanups including this little improvement shortly.

Reply:
@natoscott done!